acceptance/framework/consul/helm_cluster.go

```diff
@@ -470,3 +452,25 @@ func defaultValues() map[string]string {
 	}
 	return values
 }
+
+func CreateOrUpdateK8sSecret(t *testing.T, client kubernetes.Interface, cfg *config.TestConfig, namespace string, secretName string, secretKey string, secret string) {
```

Comment:
In Go, you can omit repeated types in function signatures which can save space on these longer lines:

```suggestion
func CreateOrUpdateK8sSecret(t *testing.T, client kubernetes.Interface, cfg *config.TestConfig, namespace, secretName, secretKey, secret string) {
```